### PR TITLE
#160609511 set up sass preprocessing

### PR DESCRIPTION
**Pull request by mendozabree**
#### What does this PR do?

Setup sass preprocessing
#### Description of Task to be completed?

Currently, our application only lets us use `.css` files for styling

This task sass allowing us to use `.scss` files for styling and creates an assets folder to hold all `.css` and `.scss` files

#### How should this be manually tested?

- Run `npm install` to install all dependencies
- Run `npm start`
- In a browser, you can view the changes that have been made using `.css` and `.scss` files 

#### What are the relevant pivotal tracker stories?

[#160609511](https://www.pivotaltracker.com/story/show/160609511)

#### Any background context you want to add?
N/A

#### Screenshots
image

### CODE REVIEW

**Review by RonKbS — APPROVED** [review]
LGTM

**Review by samuelbwambale — APPROVED** [review]
LGTM

**Comment by dondrzzy** [comment]
@mendozabree Some merge conflicts

File: src/components/landingPage/LandingPage.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
 import React from 'react';
+import '../../assets/App.scss';
+import '../../assets/App.css';
```
Comment (dondrzzy) [inline src/components/landingPage/LandingPage.js]: If this is App.css or scss, why isn't it available to the whole application. Do we have to import it in every other component as well?

**Review by dondrzzy — COMMENTED** [review]
Please refactor

**Pull request merged**